Light up LEDs on the display when switching layers

algernon opened the pull request:
When switching layers, light up the appropriate LED on the display. For this purpose, start `led_display.[ch]`, which as a start, has a function to set the brightness of a layer led.

The function will also turn all other LEDs off, and turn all of them off when on the base layer.

algernon committed:
Light up LEDs on the display when switching layers

When switching layers, light up the appropriate LED on the display. For this
purpose, start led_display.[ch], which as a start, has a function to set the
brightness of a layer led.

The function will also turn all other LEDs off, and turn all of them off when on
the base layer.

Signed-off-by: Gergely Nagy <algernon@madhouse-project.org>

mondalaci commented on right/src/keyboard_layout.c:
```diff
@@ -82,6 +82,7 @@ bool handleKey(uhk_key_t key, int scancodeIdx, usb_keyboard_report_t *report, co
         if (key_toggled_off(prevKeyStates, currKeyStates, keyId)) {
             ActiveLayer = LAYER_ID_BASE;
         }
+        LedDisplay_SetLayerLed(ActiveLayer, 0xff);
```
Let's have a global `LedDisplayBrightness` value and reference it here in place of 0xff. Later on we'll be able to reuse it elsewhere.

mondalaci commented on right/src/led_display.c:
```diff
@@ -0,0 +1,11 @@
+#define LAYER_LED_FIRST    FRAME_REGISTER_PWM_FIRST + 13
+#define LAYER_LED_DISTANCE 16
+
+void LedDisplay_SetLayerLed(uint8_t layerId, uint8_t brightness) {
```
Remove the `brightness` parameter and use `LedDisplayBrightness` in the function body according to the above.

algernon committed:
Lift out the LED brightness into a global

Instead of passing the same constant to LedDisplay_SetLayerLed() all the time,
lift it out into a global.

Signed-off-by: Gergely Nagy <algernon@madhouse-project.org>

mondalaci commented:
This is awesome! Little improvements like this one go a long way!
